CribblyBackend/Controllers/GameController.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Threading.Tasks;
+using CribblyBackend.Models;
+using CribblyBackend.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CribblyBackend.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class GameController : ControllerBase
+    {
+        private readonly IGameService GameService;
```

Comment:
🌵 📖 real nitpicky thing here, but it's nice to make private fields camelCase so it's obvious when calling it